invenio_cli/cli/cli.py

```diff
@@ -146,3 +146,14 @@ def upgrade():
     on_success = "Upgrade sucessfull."
 
     run_steps(steps, on_fail, on_success)
+
+
+@click.argument("version", nargs=1, type=str)
+@invenio_cli.command('update-app')
+def update_app(version):
+    """Updates the dependencies of your instance without upgrading to 4.0.0."""
```

Comment:
Now that we pass the version, we can remove reference to an explicit version in the helper text.

```suggestion
    """Updates the dependencies of your instance without upgrading to the next major version."""
```